bedrock/mozorg/templates/mozorg/home/includes/fundraising/takeover.html

```diff
@@ -0,0 +1,48 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+ <article class="fundraising-takeover">
+   <div class="header-strip">
+     <p><a href="{{ url('mozorg.home') }}" class="more">{{ _('Continue to mozilla.org') }}</a></p>
+   </div>
+   <section class="body">
+     <div class="content-container">
+       <header>
+         <h1>{{ high_res_img('home/takeover/wordmark.png', {'alt': 'Mozilla', 'width': '348', 'height': '92'}) }}</h1>
+         <h2>{{ _('Proudly non-profit. <br>Donate before Dec 31.') }}</h2>
+       </header>
+       <form id="fundraiser-form" method="get" action="{{ donate_url('mozillaorg_takeover') }}#amount-25">
+         <fieldset>
+           <label for="donate100">
+             <input type="radio" value="100" id="donate100" name="amount"> {{ _('$100') }}
```

Comment:
I'm not completely sure this is a good approach: are localizers supposed to know which currency we support? I think this should be determined server side.

Maybe talk with @alicoding to see how donate.mozilla.org works